com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 ### Fixed
 
+- Fixed issue where the `SetStateServerRpc` and `SetStateClientRpc` were not honoring local vs world space settings when applying the position and rotation. (#2203)
```

Comment:
Can we go with:
"Fixed issue where the **NeworkTransform** `SetStateServerRpc` and `SetStateClientRpc`", just so people reading the changelog know what this is about?